service/collector/app.go

```diff
@@ -111,9 +107,10 @@ func (c *App) collectAppStatus(ctx context.Context, ch chan<- prometheus.Metric)
 	}
 
 	for _, app := range r.Items {
-		team, ok := c.appTeamMapping[key.AppName(app)]
-		if !ok {
-			team = c.defaultTeam
+		team, err := c.getTeam(ctx, app)
+		if err != nil {
+			c.logger.Log("level", "warning", "message", fmt.Sprintf("could not get team for app %q", key.AppName(app)), "stack", fmt.Sprintf("%#v", err))
```

Comment:
```suggestion
			c.logger.Errorf(ctx, "could not get team for app %q", key.AppName(app)), err)
```
			
Why not use `c.logger.Errorf` instead?

Reply:
I like the new interface more but it would be nice to still have the level as warning.

I can live with that but we didn't update the logging here. 😱 

I'll do a follow up to update the rest.